res/controllers/Traktor-Kontrol-S2-MK3-hid-scripts.js

```diff
@@ -461,6 +465,18 @@ TraktorS2MK3.beatjumpHandler = function (field) {
     TraktorS2MK3.moveKnobEncoderState[field.group] = field.value;
 };
 
+TraktorS2MK3.activateBeatjumpHandler = function (field) {
+    if (field.value === 0) {
```

Comment:
Remove this, otherwise the rolling loop will not be exited.

Reply:
Fixed this in https://github.com/mi01/mixxx/commit/13e7ef440c9dd4d9ce332ace2afe60f4929d4351